builder/azure/common/client/config.go

```diff
@@ -109,6 +109,11 @@ func (c *Config) CloudEnvironment() *azure.Environment {
 	return c.cloudEnvironment
 }
 
+func (c *Config) AuthType() string {
+	return c.authType
+}
+
+// TODO I still need to port this to the new SDK
```

Comment:
This logic is used by every builder, I'll port is towards the end of the process and leave this todo until then